scripts/gulp/test.js

```diff
@@ -33,8 +33,32 @@ function unitTest( cb ) {
   } );
 }
 
+
+/**
+ * Run Mocha JavaScript build tests.
+ * @param {Function} cb - Callback function to call on completion.
+ */
+function buildTest( cb ) {
```

Comment:
Question for the team, should this be part of build or should 'release' really be on it's own?

Reply:
Like should it be named `releaseTest` or `buildTest` here?

Reply:
@anselmbradford yeah. We currently lint them separately, wasn't sure if we should test them separately too (if/when we ever get around to testing our build scripts). Mostly because we don't test our build files currently and naming it `buildTest` kind of implies that we are.

Reply:
I'd follow the name of the task so it's general-to-specific, so `testBuild` (also `testUnit` for the other function in the file). I think if the task is test:build, keep the function as just build.

Reply:
I'd change the task to `test:release` if we were going to change the function name. Not sure if that sways anything or if you still think release and build are fine bundled together.

Reply:
Yeah I don't have a strong opinion either way. I'd just like the function name to follow the task name.

Reply:
Decided to go with build for both linting and testing so we can bundle everything in `/scripts` in the future.